README.md

```diff
@@ -43,7 +43,7 @@ Notes:
 
 - Learn basic Bash. Actually, type `man bash` and at least skim the whole thing; it's pretty easy to follow and not that long. Alternate shells can be nice, but Bash is powerful and always available (learning *only* zsh, fish, etc., while tempting on your own laptop, restricts you in many situations, such as using existing servers).
 
-- Learn at least one text-based editor well. Ideally Vim (`vi`), as there's really no competition for random editing in a terminal (even if you use Emacs, a big IDE, or a modern hipster editor most of the time).
+- Learn at least one text-based editor well. The most common ones are Vim (`vi`), Emacs, Nano and Ed. Knowing one of these will allow you to make changes on most Linux-based systems, as these editors are present in most default setups.
```

Comment:
If we're editing the verbiage around `vim` here, I'd suggest against referring to `vi` and `vim` as if they are one and the same.